address.js

```diff
@@ -31,6 +31,7 @@ define(function(require) {
         , callback
         , target
         , into
+        , timeout = 30
```

Comment:
See my comment below about exporting this value as a constant from the middleware instead.